lms/migrations/versions/9e79650bed37_backfill_lms_course_application_instance.py

```diff
@@ -0,0 +1,43 @@
+"""LMSCourseApplicationInstance backfill."""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "9e79650bed37"
+down_revision = "e13fb37c96e5"
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+    conn.execute(
+        sa.text(
+            """
+        WITH backfill as (
+            SELECT
+                "grouping".created,
+                "grouping".updated,
+                lms_course.id lms_course_id,
```

Comment:
Is there an `AS` missing here?

Reply:
No, `AS` is optional but arguably the query would be more clear with it.